web/js/map-OpenLayers.js

```diff
@@ -115,6 +115,67 @@ var fixmystreet = fixmystreet || {};
         fixmystreet.markers.redraw();
       },
 
+      shortlist_multiple: function(ids, token) {
+        $.post("/my/planned/change_multiple", { ids: ids, token: token })
+        .done(function() {
+          var $itemList = $('.item-list'),
+              items = [];
+          $('.shortlisted-status').remove();
+
+          for (var i = 0; i < ids.length; i++) {
+            var problemId = ids[i],
+                $item = $itemList.find('#report-'+ problemId)
+                $form = $item.find('form'),
+                $submit = $form.find("input[type='submit']" );
+
+            $item.attr('data-lastupdate', new Date().toISOString());
```

Comment:
I'm just following the conventions that were applied on a single update a la: https://github.com/mysociety/fixmystreet/blob/master/web/cobrands/fixmystreet/offline.js#L413-L416